js/ftx.js

```diff
@@ -1378,11 +1378,30 @@ module.exports = class ftx extends Exchange {
         //         "type":"limit"
         //     }
         //
+        // fetchOrder (condition-order)
+        //
+        //     {
+        //         "time": "2022-03-29T04:54:04.390665+00:00",
+        //         "orderId": 132144259673,  //this is not the same id used in fetchOrder, instead reference id
```

Comment:
A side-note on formatting, please, always put one space (not two) before the opening `//` comment, and one space after. Keep the formatting uniform across the file and across the exchanges.

```JavaScript
"orderId": 12345678,  //this is not the same id used in fetchOrder, instead reference id
↓
"orderId": 12345678, // this is not the order id used in fetchOrder, it is a reference id
```